src/gui/folder.cpp

```diff
@@ -46,6 +45,9 @@
 
 namespace OCC {
 
+// Start with "sync." since this is mostly sync logic at the wrong layer
```

Comment:
i disagree with that.   it is scheduling logic,  but not sync logic.

But that does not matter much.